core/templates/pages/exploration-player-page/exploration-player-page.component.html

```diff
@@ -3,10 +3,29 @@
     <oppia-loading-message [message]="'Loading'"></oppia-loading-message>
   </div>
   <background-banner *ngIf="!isLoadingExploration"></background-banner>
+  <div *ngIf="explorationIsUnpublished" class="unpublished-exploration-banner">
```

Comment:
```suggestion
  <div *ngIf="explorationIsUnpublished" class="oppia-unpublished-exploration-banner">
```
Please prepend class names with `oppia-`.

Reply:
Done.